java/parser.js

```diff
@@ -1,171 +1,206 @@
 'use strict'
 const types = require('./mapping/types')
 const selectors = require('./mapping/selectors')
-const {capitalizeFirstLetter} = require('./util')
+const {capitalizeFirstLetter, isEmpty} = require('./util')
+const {CHECK_SETTINGS_HOOKS, CHECK_SETTINGS_OPTIONS} = require('./mapping/supported')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
const {CHECK_SETTINGS_HOOKS, CHECK_SETTINGS_OPTIONS} = require("./mapping/supported")
```